README.md

```diff
@@ -72,19 +81,22 @@ to **v1**
 
 1. To start the program, run:
 
-    ```shell
+    ```sh
     ./cloud_run_release_manager -cli -project=<YOUR_PROJECT>
     ```
 
 Once you run this command, it will check the health of Cloud Run services with
 the label `rollout-strategy=gradual` every minute by looking at the candidate's
 metrics for the past 30 minutes by default.
 
-- The health is determined using the metrics and configured health criteria
-- By default, the only health criteria is a expected max server error rate of
+* The health is determined using the metrics and configured health criteria
+* The Release Manager expects `100` requests in the past 30 minutes before
```

Comment:
We actually need to rethink this since, as pointed by glasnt, this would require a lot of requests to the service to be met